src/Tasks/Microsoft.NET.Build.Tasks/ResolveReadyToRunCompilers.cs

```diff
@@ -257,6 +257,11 @@ private static bool ExtractTargetPlatformAndArchitecture(string runtimeIdentifie
                 case "arm64":
                     architecture = Architecture.Arm64;
                     break;
+#if !NETFRAMEWORK
+                case "riscv64":
+                    architecture = Architecture.RiscV64;
```

Comment:
It is not needed for VS at this point. This is just an enabler and we are only focusing on linux-riscv64 bring up. We can add more stuff once later on.

Reply:
In my experience these sorts of updates tend to be forgotten later on :)

Reply:
* `(Architecture)9` sounds like it will be supported by .NET Framework someday (which is not happening since .NET Framework is on its discontinuation path).
* riscv64 is currently a community-supported platform and port is still in progress. VS support for linux-riscv64 is not even on the horizon.

By the time it will be, MSBuild for VS may even switch to netcoreapp. If you have different outlook on the matter, I can add it, no problem. :)

Reply:
> _crossgenTool 

Old crossgen is discontinued (in favor of crossgen2). riscv64 doesn't have crossgen (1) support.

Reply:
> (Architecture)9 sounds like it will be supported by .NET Framework someday (which is not happening since .NET Framework is on its discontinuation path).

It's not really about .NET Framework since these are just build tasks. But yeah, we can add it later if necessary.